styles.css

```diff
@@ -0,0 +1,340 @@
+/*niet goed natuurlijk, daarnaast lukte het niet om de container niet te laten groeien/krimpen.*/
+/*Ik heb geprobeert om van de container een item te maken en vervolgens grow/shrink op nul, maar*/
+/*dat werkte niet. Zou wel moeten werken denk ik, weet niet wat ik fout deed en was te laat voor SME*/
+/*Enne nu je dit toch aan het lezen bent, heb het wel gezien in de opdracht maar het is zoals je*/
+/*kunt zien niet gelukt om weg te blijven van  letterlijke height: 500px properties*/
+.flex-container-section3 {
+    display: flex;
+    flex-direction: row;
+    min-height: 450px;
+    flex-wrap: wrap;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    column-gap: 30px;
+    row-gap: 15px;
+    justify-content: center;
+}
+.outer-col-section3 {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    height: 200px;
+    border: 1px solid #5722CD;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.2);
+    flex-shrink: 0;
+    flex-grow: 0;
+    padding: 15px;
+}
+.img-section3 {
+    width: 50px;
+    height: 50px;
+}
+.outer-col-section3 h4{
+    color: #5722CD;
+    padding-bottom: 10px;
+    margin-top: 5px;
+}
+.outer-col-section3 p {
+    color: #7A7A7A;
+    font-size: 9px;
+}
+/* ---------------3.6 SECTION 3 ------------------- */
+form {
+    min-height: 450px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+
+    color: white;
+    font-family: inherit;
+    background-color: #0066ff;
+    font-size: 12px;
+}
+.fieldset {
+    display: flex;
+    flex-direction: column;
+}
+textarea {
+    border-radius: 2px;
+    font-family: inherit;
+    padding: 8px;
+    outline: none;
+}
+form p {
+    font-size: 10px;
+}
+form h2 {
+    font-size: 30px;
+    margin: 20px;
+}
+.input-tag{
+    border-radius: 2px;
+    border: none;
+    padding: 8px;
+    outline: none;
+}
+label {
+    margin-top: 15px;
+}
+.button-form {
+    background-color: #ED4385;
+    color: white;
+    border-radius: 50px;
+    text-transform: uppercase;
+    font-size: 8px;
+    border: none;
+    min-width: 90px;
+    min-height: 32px;
+    margin-top: 15px;
+    margin-bottom: 30px;
+}
+/* --------------- 3.7 F O O T E R------------------- */
+footer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: black;
+    color: white;
+    height: 40px;
+    font-family: inherit;
+    font-size: 10px;
+}
+/* ------------------------------- 4. Media Queries -------------------------------- */
+
+@media screen and (max-width: 550px) {
+    /*---------------------------------------H E A D E R----------------------*/
+    .text-header {
+        max-width: 90%;
+    }
+
+    .text-p {
+        font-size: 25px;
+    }
+
+    .button-header {
+        padding: 24px 40px;
+        font-size: 20px;
+    }
+
+    /*---------------------------------------S E C T I O N 1 ----------------------*/
+    .flex-container-section1 {
```

Comment:
De sections containers die je gebruikt lijken vrij veel op elkaar, maar omdat jij een `min-height` gebruikt en geen padding of margin, moet je dus voor elke container een eigen class naam maken. Dit had allemaal onder één classe gekunt wanneer je padding of margin had toegepast.